test/jdk/java/awt/PopupMenu/PopupMenuLocation.java

```diff
@@ -44,23 +51,26 @@
  */
 public final class PopupMenuLocation {
 
-    private static final int SIZE = 350;
     public static final String TEXT =
             "Long-long-long-long-long-long-long text in the item-";
-    private static volatile boolean action = false;
+    private static final int SIZE = 350;
+    private static Robot robot;
+    private static volatile CountDownLatch actionEventReceivedLatch;
 
     public static void main(final String[] args) throws Exception {
+        robot = new Robot();
+        robot.setAutoDelay(200);
         GraphicsEnvironment ge =
                 GraphicsEnvironment.getLocalGraphicsEnvironment();
         GraphicsDevice[] sds = ge.getScreenDevices();
         for (GraphicsDevice sd : sds) {
             GraphicsConfiguration gc = sd.getDefaultConfiguration();
             Rectangle bounds = gc.getBounds();
-            Point point = new Point(bounds.x, bounds.y);
+            Point point = new Point(bounds.x + 20, bounds.y + 20);
```

Comment:
Why do you need to add this additional 20 pixels? When we click on the frame we already shift the mouse click location by 130 pixels from the top of the frame.

Reply:
Reverted it